.github/workflows/cmake.yml

```diff
@@ -87,27 +87,9 @@ jobs:
         # Execute the build.  You can specify a specific target with "--target <NAME>"
         run: cmake --build . --config $BUILD_TYPE
 
```

Comment:
nit: maybe we want to make these matrix builds?

Reply:
For multiple OS?